tests/ui_tests/test_service_page.py

```diff
@@ -0,0 +1,403 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""UI tests for /service page"""
+import os
+
+import allure
+import pytest
+from _pytest.fixtures import SubRequest
+from adcm_client.objects import (
+    ADCMClient,
+    Bundle,
+    Provider,
+)
+from adcm_pytest_plugin import params
+from adcm_pytest_plugin import utils
+
+from tests.library.status import ADCMObjectStatusChanger
+from tests.ui_tests.app.page.admin.page import AdminIntroPage
+from tests.ui_tests.app.page.cluster.page import (
+    ClusterServicesPage,
+)
+from tests.ui_tests.app.page.common.import_page.page import (
+    ImportItemInfo,
+    SUCCESS_COLOR,
+    NEGATIVE_COLOR,
+)
+from tests.ui_tests.app.page.common.status.page import StatusRowInfo
+from tests.ui_tests.app.page.service.page import (
+    ServiceComponentPage,
+    ServiceStatusPage,
+)
+from tests.ui_tests.app.page.service.page import (
+    ServiceMainPage,
+    ServiceConfigPage,
+    ServiceImportPage,
+)
+
+BUNDLE_COMMUNITY = "cluster_community"
+CLUSTER_NAME = "Test cluster"
+SERVICE_NAME = "test_service"
+PROVIDER_NAME = 'test_provider'
+HOST_NAME = 'test-host'
+PROVIDER_WITH_ISSUE_NAME = 'provider_with_issue'
+COMPONENT_NAME = "first"
+BUNDLE_REQUIRED_FIELDS = "cluster_and_service_with_required_string"
+BUNDLE_IMPORT = "cluster_to_import"
+
+# pylint: disable=redefined-outer-name,no-self-use,unused-argument
+pytestmark = pytest.mark.usefixtures("login_to_adcm_over_api")
+
+
+@pytest.fixture()
+def create_community_cluster(sdk_client_fs: ADCMClient):
+    """Create community edition cluster"""
+    bundle = cluster_bundle(sdk_client_fs, BUNDLE_COMMUNITY)
+    return bundle.cluster_create(name=CLUSTER_NAME)
+
+
+@pytest.fixture()
+def create_community_cluster_with_service(sdk_client_fs: ADCMClient):
```

Comment:
When you return more than 1 object, please add return type

Reply:
add hints